notesapi/v1/serializers.py

```diff
@@ -56,7 +56,7 @@ def get_text(self, note):
         Return note text.
         """
         if note.highlighted:
-            return note.highlighted[0].decode('unicode_escape')
```

Comment:
I wasn't involved in this work, so I don't know. It does seem like a strange thing to be doing. @jimabramson do you have any insights?